src/core/SablierLockup.sol

```diff
@@ -108,6 +111,17 @@ contract SablierLockup is ISablierLockup, SablierLockupBase {
         tranches = _tranches[streamId];
     }
 
+    /// @inheritdoc ISablierLockup
+    function getUnlockAmounts(uint256 streamId)
+        external
+        view
+        override
+        notNull(streamId)
+        returns (LockupLinear.UnlockAmounts memory unlockAmounts)
+    {
+        unlockAmounts = _unlockAmounts[streamId];
```

Comment:
It should revert if the model is not Lockup Linear.

Reply:
yes, waiting for the last rebase

Reply:
I have already rebased from #1069 this morning.